### DISCUSSION TITLE
Multiple line notes not working

### DISCUSSION THREAD

**Discussion by flaviuvlaicu** [CorentinTh/enclosed#228] [General]
Hi,
just wanted to let you know that if you for example want to paste some markdown text, it does not keep the format. Also for example : 
If you want
To paste some
Text
on 
each line :) 

It does not work.

**Comment by CorentinTh** [comment]
Thanks for pointing that out! Yeah, we definitely need to wrap the note display in a `pre` tag to preserve those new lines. It should be an easy fix

By the way, for issues like this in the future, it would be nice if you could open a bug report in the issues section instead of a discussion, it'll help us keep track of things more easily

**Reply by CorentinTh** [reply]
Hi @flaviuvlaicu fixed in #229

**Reply by flaviuvlaicu** [reply]
Is it difficult to add code highlights in the post? for example bash, python etc?
